api/proto/auth/authproto.proto

```diff
@@ -204,7 +204,7 @@ message CreateResetPasswordTokenRequest {
     // Type is a token type.
     string Type = 2 [ (gogoproto.jsontag) = "type" ];
     // TTL specifies how long the generated token is valid for.
-    int64 TTL = 3 [ (gogoproto.jsontag) = "ttl", (gogoproto.casttype) = "Duration" ];
+    int64 TTL = 3 [ (gogoproto.jsontag) = "ttl", (gogoproto.casttype) = "time.Duration" ];
```

Comment:
Can you do some code archaeology and find out why we were using `Duration` here.

Reply:
The custom type is simply `time.Duration` with an extra `Get` and `Set` method, so I don't think it's necessary. I think it was created to model the duration import in `/lib/services/types.proto`, but that custom duration actually has marshalling logic added on. that's why it has the inaccurate comment from that services.Duration:
```// Duration is a wrapper around duration to set up custom marshal/unmarshal```. 
I pinged Alexey about it below in `proto.go`.